### ISSUE TITLE
Add communities to the global search

### ISSUE THREAD

**Issue by ewan-escience** [research-software-directory/RSD-as-a-service#1248]
Even when communities are enabled, the global search bar only searches for software, projects and communities. We should also search for communities.

**Comment by dmijatovic** [comment]
Yes, in addition with introduction of modules in the settings.json we should adjust search to only filter on the enabled modules: software, projects, organisations, communities.
